tests/odr/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+file(GLOB_RECURSE FOUND_HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/../../include/*.hpp")
+list(TRANSFORM FOUND_HEADERS REPLACE ".*/include/" "")
+list(FILTER FOUND_HEADERS EXCLUDE REGEX ".*dnsa_connection.*")
```

Comment:
These magic rules are fragile, please at least factor this out into something that explicitly mentions "list of  headers covered by test-extended"

Reply:
Besides, do we have a good reason to leave them out for ODR test?

Reply:
They require boost, it was also in makefile this way afaik

Reply:
True, it was there that way. And since the original ODR test didn't consider extended tests, I wondered if it would be worth it to add it (it would require to add boost here).

Reply:
Removed and added boost link.